lib/parse.js

```diff
@@ -65,23 +65,36 @@ module.exports = (input) => {
   }
 
   return _.compact(_.map(input.split(/\n\s*\n/), (device) => {
-
     device = _.chain(device)
       .split('\n')
       .filter((line) => {
         return /^(\s\s-\s)?[a-z]+:/g.test(line);
       })
       .map((line) => {
-        return line.replace(/"/g, (match, index, string) => {
-          if (_.some([
-            string.indexOf('"') === index,
-            string.lastIndexOf('"') === index
-          ])) {
-            return match;
-          }
+        return line
+          .replace(/\t/g, '    ')
+          .replace(/\\[^.\\]/g, (match, index, string) => {
+            if (string[index - 1] === '\\') {
+              return match;
+            }
+
+            return _.last(match);
+          })
+
+          // Remove non printable ascii characters
+          // See http://stackoverflow.com/a/24229554
+          .replace(/[^\x20-\x7E]+/g, '')
```

Comment:
Heh, very similar to the regex at https://github.com/resin-io/etcher/blob/master/scripts/ci/ensure-all-text-files-only-ascii.sh#L31 ;-)